devlab/config.template

```diff
@@ -0,0 +1,137 @@
+# Do not modify. This is template file. To change parameters please use 'config.ini'.
+
+[migrate]
+keep_user_passwords = False
+ssh_transfer_port = 9990-9999
+key_filename = <migrate_key_filename>
+keep_ip = yes
+keep_floatingip = yes
+keep_volume_storage = False
+speed_limit = 100MB
+instances = key_name-qwerty
+file_compression = gzip
+level_compression = 9
+overwrite_user_passwords = False
+migrate_quotas = False
+direct_compute_transfer = yes
+filter_path = configs/filter.yaml
+keep_lbaas = no
+ssh_chunk_size = 100
+retry = 5
+migrate_extnets = True
+time_wait = 5
+
+[mail]
+server = <server_name:port_number>
+username = <username>
+password = <password>
+from_addr = <e-mail address>
+
+[src]
+type = os
+auth_url = http://<grizzly_ip>:35357/v2.0
+host = <grizzly_ip>
+ssh_host = <grizzly_ip>
+ext_cidr = <src_ext_cidr>
+user = <src_user>
+password = <src_password>
+tenant = <src_tenant>
+temp = <src_temp>
+
+[src_mysql]
+user = <src_mysql_user>
+password = <src_mysql_password>
+host = localhost
+connection = mysql+mysqlconnector
+
+[src_rabbit]
+password = <src_rabbit_password> 
+hosts = <grizzly_ip>
+
+[src_compute]
+service = nova
+backend = iscsi
+convert_diff_file = qcow2
+convert_ephemeral_disk = qcow2
+host_eph_drv = <grizzly_ip>
+
+[src_storage]
+service = cinder
+backend = ceph
+host = <grizzly_ip>
+# protocol_transfer=GLANCE ; or SSH
+protocol_transfer = src_storage_protocol_transfer 
+disk_format = qcow2
+# Ceph
+# volume_name_template=volume-
+# rbd_pool=volumes
+
+[src_image]
+service = glance
+backend = swift
+
+[src_identity]
+service = keystone
+
+[src_network]
+service = auto
+interfaces_for_instance = <src_network_interfaces_for_instance>
+
+[src_objstorage]
+service = swift
```

Comment:
We should pass it blank, because of no swift on our dev environments. Also we should add the same section for dst.